kannada/kannada_kali/kannada_kali.md

```diff
@@ -0,0 +1 @@
+ಕನ್ನಡ ಕಲಿಯಲು [ಇದು](https://play.google.com/store/apps/details?id=com.pgjpb.learnkannada&hl=en)  ಒಳ್ಳೆಯ ಸಾಧನ.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.